README.md

```diff
@@ -1 +1 @@
-# quantified-self-be
+# Quantified Self Be
```

Comment:
@jplao this is just a silly merge conflict please feel free to solve it and merge to master

Reply:
@Diazblack conflict resolved and merged.